### DISCUSSION TITLE
Caprover installation sucessfull but I cant login.

### DISCUSSION THREAD

**Discussion by 546451cc** [chaskiq/chaskiq#891] [Q&A] [answered]
Hi there I make a caprover installation and all seems ok, but when I try to login show "error" but don´t specific the reason.
Also in Sidekiq I added chaskiq env variables. Thats correct? or are other kind env variables.
Here is a screenshot. Thanks!
![image](https://user-images.githubusercontent.com/53021603/211213090-68ddba75-0cfc-45fb-a1b5-7ad42922085f.png)

**Comment by michelson** [comment, marked as answer]
Did you set the ADMIN_EMAIL and ADMIN_PASSWORD and run the admin_generator ?

**Reply by 546451cc** [reply]
I think was that, I don´t run the admin_generator but finally I make a 1 click install and works fine. But I have a problem to show the snippet. When I go to "get the snippet" and them copy the code don´t show the widget chat, In chat tester don´t show too. I check in Messenger settings / aparance settings and I check "Activate messenger" and "Display new messages in floating box" but don´t work, sorry to ask another question, do you have any Idea whats happen?

**Comment by michelson** [comment]
Hi, you need to add the domain to the domain_url field on config -> app settings config

The domain is the url of the site where the embed will be on.

**Reply by 546451cc** [reply]
Oh, thanks! also works with subdomains?

**Comment by MISanjaI** [comment]
Sir, I'm facing the issue? I couldn't able to figure out what to do next !!! can you help me to fix this ?? please ... !!!
